pallets/pips/src/lib.rs

```diff
@@ -1168,6 +1168,7 @@ impl<T: Config> Module<T> {
                         T::UpgradeCommitteeVMO::ensure_origin(origin).map(|_| Committee::Upgrade)
                     })
                     .map(Proposer::Committee)?;
+                // TODO: I think we should use a fixed DID for each of the committees.
                 let did = Context::current_identity::<Identity<T>>()
                     .ok_or_else(|| Error::<T>::MissingCurrentIdentity)?;
                 Ok((proposer, did))
```

Comment:
The `origin` here is one of the committees.  Right now `current_identity` is the DID of the committee member that made the final vote (which directly executes the call to propose a pip).  We can either use GC_DID or a fixed DID for each committee.

Reply:
Can we create a "dynamic" programmatic DID for each committee (since they are configured "dynamically")?

Reply:
These committees are fixed in the runtime, so for here at least the easiest change would be to generate named DIDs like we do for `GC_DID`.  If we want to define the committee DID in each pallet, then we could create a `CommitteeOrigin` trait and use that instead of the `FromOrigin` trait being used here.